src/Applications/GEOSctm_App/CMakeLists.txt

```diff
@@ -38,12 +38,21 @@ install (
    DESTINATION etc
    )
    
+# Did we build with Hydrostatic?
 if(HYDROSTATIC)
    set(CFG_HYDROSTATIC TRUE)
 else()
    set(CFG_HYDROSTATIC FALSE)
 endif()
 
+# Did we build on Rome?
```

Comment:
In contrast, this comment is helpful as "EPYC" is a bit obscure.   But for the non HW people maybe "Rome hardware" or "Rome processor"  (even I am not quite clear on what Rome actually means).

Reply:
Okay. I'll make some changes and have you review.

Reply:
Agree - not worth impacting all other repos.  And I think it is obvious enough as is in context.